packages/mui-base/src/PopperUnstyled/PopperUnstyled.js

```diff
@@ -78,13 +78,14 @@ const PopperTooltip = React.forwardRef(function PopperTooltip(props, ref) {
   }, [handlePopperRef]);
   React.useImperativeHandle(popperRefProp, () => popperRef.current, []);
 
+  const resolvedAnchorElement = resolveAnchorEl(anchorEl);
```

Comment:
@michaldudak Is this what you asked for?

Reply:
No, I meant:
```diff
-const [tooltipAnchorEl, setTooltipAnchorEl] = React.useState(resolveAnchorEl(anchorEl));
+const [resolvedAnchorElement, setResolvedAnchorElement] = React.useState(resolveAnchorEl(anchorEl));
```